packages/dataviews/src/view-table.js

```diff
@@ -545,6 +545,9 @@ function ViewTable( {
 		// TODO:Add spinner or progress bar..
 		return <h3>{ __( 'Loading' ) }</h3>;
 	}
+
+	const sortValues = { asc: 'ascending', desc: 'descending' };
```

Comment:
Don't know if there's a general preference for raw objects or Maps, but happy to change this as necessary...

```suggestion
	const sortValues = new Map([ ['asc', 'ascending'] , ['desc', 'descending'] ]);
```